lab-matt/lib/second.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Again this looks great, but is essentially the same as your first.js function.  It is better to not repeat your code in this manner and instead create a module which would perform essentially the same function, but instead take an additional parameter to determine what file to lookup. (For ex: readFile(file, callback) { ... })